README.md

```diff
@@ -35,12 +35,38 @@ Email: info@iliosproject.org
 
 # Development of Ilios
 
-You can build a run a local version of Ilios easily using [docker](https://www.docker.com/).
-There are convenient clients for Mac and Windows and instructions Linux.
+The Ilios API is a PHP application which required a MySql server.  The easiest way to get a working
+MySql database running is by using the ilios/mysql-demo [docker](https://www.docker.com/) image.
+After installing docker you can start a demo database by running:
 
-After you have installed docker on your machine you can startup a local version by changing into the Ilios directory and running:
 ```bash
-docker-compose up
+docker run -d --name db -p 3306:3306 ilios/mysql-demo
 ```
 
-You will now be able to access ilios at http://localhost:8000
+Now modify your `/app/config/parameters.yml` to connect to this database:
+
+```yaml
+parameters:
+    database_name: ilios
+    database_driver: pdo_mysql
+    database_host: 127.0.0.1
+    database_port: null
+    database_user: ilios
+    database_password: ilios
+    database_mysql_version: 5.7
+```
+
+You can test that your database is configured correctly by running:
+
+```bash
+bin/console doctrine:schema:validate
+```
+
+Now you can start a PHP development webserver by running:
+
+```bash
+ILIOS_API_ENVIRONMENT=dev ILIOS_API_DEBUG=true bin/console server:start --router=web/app.php
+```
+
+
+You will now be able to access Ilios at http://localhost:8000
```

Comment:
You should now be able to access your newly-Dockerized instance of Ilios by visiting http://localhost:8000 in you browser.